wiki/osu!dev_Discord_server/fr.md

```diff
@@ -0,0 +1,33 @@
+---
+tags:
+  - développement
```

Comment:
please add tags from the original article

Reply:
I don't see any other tag on the original article, am I missing something ?

Reply:
`development` it is, as opposed to `développement` -- you are free to keep either the original one, or both if you want this article to also rank higher in search results for `development` and `développement`